DOL.WHD.Section14c.Web/src/modules/components/attachmentField/attachmentFieldController.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function(ngModule) {
+    ngModule.controller('attachmentFieldController', function($scope, apiService, stateService) {
+        'ngInject';
+        'use strict';
+
+        var vm = this;
+        vm.attachmentApiURL = apiService.attachmentApiURL + stateService.ein;
+        vm.access_token = stateService.access_token;
+        
+        this.onAttachmentSelected = function(fileinput) {
+            if(fileinput.files.length > 0){
+                apiService.uploadAttachment(stateService.access_token, stateService.ein, fileinput.files[0]).then(function (result){
+                    $scope.attachmentId = result.data[0].id;
```

Comment:
The API is setup to handle multiple files, but this implementation always uploades one file.  On Ln14 `fileinput.files[0]` always only sends one file.  So the API will always return the results of the one file upload or an error.